src/Panel/Dialog.php

```diff
@@ -50,6 +50,12 @@ public static function routes(
 		$pattern = trim($prefix . '/' . ($options['pattern'] ?? $id), '/');
 		$type    = str_replace('$', '', static::$key);
 
+		// create load/submit events from class handler
+		if ($handler = $options['handler'] ?? null) {
```

Comment:
`Discuss:` We weren't sure about the term `handler`.

Reply:
First thought while looking at it again, do we want to use Controller here?

Reply:
Yes, that was my original thought as well, but we also started using the same term for the API (see Changes) and the Panel (see Search) with classes.

Reply:
Hmm yes, with Search is true. But would it be weird that then some controllers are located in `Panel\Controllers`, others though we pull in from `Panel\Ui\Dialogs`?